libc/spec/llvm_libc_ext.td

```diff
@@ -51,8 +51,21 @@ def LLVMLibcExt : StandardSpec<"llvm_libc_ext"> {
       ]
   >;
 
+  HeaderSpec Math = HeaderSpec<
+      "math.h",
```

Comment:
The declarations here weren't sorted alphabetically (`Strings`, then `Assert`, then `Sched`), so I added `Math` at the end. The `Headers` list below was sorted alphabetically, so I kept it sorted.